### PR TITLE
Michael Foster staff page

### PR DESCRIPTION
**Pull request by jmafoster1**
Added my profile page

### CODE REVIEW

**Review by ptheywood — APPROVED** [review]

File: _people/michael-foster.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+othernames: Michael   
+surname: Foster
+role: Research Software Engineer
+
+---
+
+Michael joined the RSE team in October 2024.
+He comes from a background in Computer Science, following an integrated masters degree and PhD in software testing at the University of Sheffield.
+After that, he was a research associate on the [CITCoM project](https://sites.google.com/sheffield.ac.uk/citcom/home) in the software testing group, also at the University of Sheffield.
+During this time, he has been an active member of the software engineering research community and an advocate for good RSE practice, having reviewed software artefacts for several software engineering conferences.
```
Comment (Robadob) [inline _people/michael-foster.md]:
```suggestion
During this time, he has been an active member of the software engineering research community and an advocate for research software best practices, having reviewed software artefacts for several software engineering conferences.
```

Maybe

Comment (Robadob) [reply _people/michael-foster.md]: I'm not 100% sure what would be considered a "software artefact", so I'd guess any non-RSE will be even more confused. Perhaps clarify?

**Review by Robadob — APPROVED** [review]
Looks good in principle, see my comment for my only suggestion but feel free to ignore.

File: _people/michael-foster.md (hunk 1 above)
Comment (ns-rse) [reply _people/michael-foster.md]: I actually prefer "good practice(s)" over "best practice(s)" as the later subtley implies there isn't room for improvement where as the former is more open to improvement. But that is my pedantic view.

> what would be considered a "software artefact"

Lots of debugging output! :laughing:

Comment (Robadob) [reply _people/michael-foster.md]: Tbh my initial suggestion was that RSE should be replaced with research software, I then went further to replace with "best practices" as that's the generic terminology used by everyone. It's not to suggest the best practices are fixed and not still evolving.

Comment (jmafoster1) [reply _people/michael-foster.md]: Thanks for the suggestions. In the end, I changed to "good research software practice" to try to combine the best of both. I also removed the term "software artefacts" to try to clarify things.

**Pull request merged**